firmware/boron-ins-fsm/src/imDoorSensor.cpp

```diff
@@ -58,6 +59,32 @@ void initializeDoorID() {
     }
 }
 
+void initializeAntennaType() {
+    uint16_t initializeBLEAntennaTypeFlag;
+
+    // Boron flash memory is initialized to all F's (1's)
+    EEPROM.get(ADDR_INITIALIZE_BLE_ANTENNA_TYPE_FLAG, initializeBLEAntennaTypeFlag);
+    Log.info("Antenna type flag is 0x%04X", initializeBLEAntennaTypeFlag);
+
+    if (initializeBLEAntennaTypeFlag != INITIALIZE_ANTENNA_TYPE_FLAG) {
+        bool externalBLEAntennaDefault = false;
+        EEPROM.put(ADDR_BLE_ANTENNA_TYPE, externalBLEAntennaDefault);
+        initializeBLEAntennaTypeFlag = INITIALIZE_ANTENNA_TYPE_FLAG;
+        EEPROM.put(ADDR_INITIALIZE_BLE_ANTENNA_TYPE_FLAG, initializeBLEAntennaTypeFlag);
+        Log.info("BLE antenna type was written to flash on bootup.");
+    }
+    else {
+        EEPROM.get(ADDR_BLE_ANTENNA_TYPE, usingExternalBLEAntenna);
+        if (usingExternalBLEAntenna == true) {
```

Comment:
Because `usingExternalBLEAntenna` is a boolean, so it is more idomatic to leave off the `== true` and just have `if (usingExternalBLEAntenna) {`